packages/core/src/common/workspace-utils.ts

```diff
@@ -0,0 +1,39 @@
+/********************************************************************************
+  * Copyright (C) 2018 Ericsson and others.
+  *
+  * This program and the accompanying materials are made available under the
+  * terms of the Eclipse Public License v. 2.0 which is available at
+  * http://www.eclipse.org/legal/epl-2.0.
+  *
+  * This Source Code may also be made available under the following Secondary
+  * Licenses when the conditions for such availability set forth in the Eclipse
+  * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+  * with the GNU Classpath Exception which is available at
+  * https://www.gnu.org/software/classpath/license.html.
+  *
+  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+  ********************************************************************************/
+
+import { Path } from './path';
+import URI from './uri';
+
+export namespace WorkspaceUtils {
```

Comment:
core should not have a mention of a workspace, there is a workspace package for it

Reply:
I agree and the implementation does not require the context of a workspace anymore, perhaps I can move it into a new file called `path-utils` or I can directly add the function to the `path` file.

Reply:
`Path` is a set of generic functions to work with paths, this function seems to only make sense in the context of the filesystem. I would move it at least there.